tests/test_dswx_hls_workflow.py

```diff
@@ -91,4 +91,4 @@ def test_workflow():
 
         assert compare_dswx_hls_products(ref_file, output_file)
 
-    assert True
\ No newline at end of file
+    assert True
```

Comment:
Out of curiosity, what is the reason for an `assert True` at the end? This should always pass; is it something to do with the test environment? Thanks!